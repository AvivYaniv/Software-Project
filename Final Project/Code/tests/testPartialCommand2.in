solve tests/b2_2.txt
set 1 3  4