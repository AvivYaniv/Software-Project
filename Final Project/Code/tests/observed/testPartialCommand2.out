Sudoku
------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|  4     |      2.|
|        |        |
-------------------
Enter your command:
Exiting...
